client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lebkuchen-fm-client",
   "private": true,
-  "version": "2025.2.0",
+  "version": "2025.3.0",
```

Comment:
I wonder if it's major or minor version 🤔 @gorzalczany what do you think? `2025.3.0` or `2025.2.1`?

Reply:
Hard question. It would be a really good use case for feature/release branch, but at the same time there is no point in freezing this code. I think that it is a feature, even if it is "just" a feature for developers so 2025.3.0 sounds good to me.

Reply:
`2025.3.0` it is then 👍